arviz/numeric_utils.py

```diff
@@ -1,11 +1,24 @@
 """Numerical utility functions for ArviZ."""
 import warnings
 import numpy as np
-from scipy.signal import convolve, convolve2d
+from scipy.signal import convolve2d
 from scipy.sparse import coo_matrix
 
 from .stats.stats_utils import histogram
 from .utils import _stack, _dot, _cov
+from .kde_utils import _kde
```

Comment:
I think this is a cyclic import

Reply:
I moved things from kde_utils.py to numeric_utils.py (as suggested) but that gives us this cyclic import. It works well because I don't use a relative import in kde_utils, but I don't feel it is very elegant/proper. Any suggestions on how to improve it?

Reply:
given that `_fast_kde` is a private method, I think we can leave it in `numeric_utils` raising an error and pointing to `_kde`

Reply:
I am not sure if we should have `_kde` available again as `az._kde` though

Reply:
I think we should as we used to have _fast_kde